CMakeLists.txt

```diff
@@ -337,7 +337,10 @@ IF(NOT CYCLUS_DOC_ONLY)
 
         # make sure we know about having python
         add_definitions(-DCYCLUS_WITH_PYTHON)
-        set(LIBS ${LIBS} ${PYTHON_LIBRARIES})
+        IF (NOT APPLE)
+            MESSAGE("ININININHBJGVHGCVGFCGFCGFCGFCGFCGFCGFCGFC:")
```

Comment:
Is this necessary?? or something leftover from debugging?

Reply:
pushy sorry, forgot to clean this up....